peekingduck/pipeline/nodes/draw/bbox.py

```diff
@@ -22,7 +22,26 @@
 
 
 class Node(AbstractNode):
-    """Draw node for drawing bboxes onto image"""
+    """Draw bounding boxes on image.
+
+    The draw bbox node uses the bboxes and, optionally, the bbox labels from the model
+    predictions to draw the bbox predictions onto the image.
+    For better understanding of the usecase, refer to the object counting usecase.
```

Comment:
link to object_counting page via the following format:

```suggestion
    For better understanding of the usecase, refer to the `object counting usecase <use_cases/object_counting.html>`_.
```

Reply:
Changed according to suggestion